pkg/datapath/loader/base.go

```diff
@@ -368,11 +369,11 @@ func (l *loader) reinitializeXDPLocked(ctx context.Context, extraCArgs []string,
 // ReinitializeXDP (re-)configures the XDP datapath only. This includes recompilation
 // and reinsertion of the object into the kernel as well as an atomic program replacement
 // at the XDP hook. extraCArgs can be passed-in in order to alter BPF code defines.
-func (l *loader) ReinitializeXDP(ctx context.Context, cfg *datapath.LocalNodeConfiguration, extraCArgs []string) error {
+func (l *loader) ReinitializeXDP(ctx context.Context, cfg *datapath.LocalNodeConfiguration, extraCArgs []string, xdpConfig xdp.Config) error {
```

Comment:
Can we make the xdpConfig a field of `datapath.LocalNodeConfiguration` please? We are trying to move all external configuration into this one structure.

Reply:
Done!